src/test/kotlin/net/maizegenetics/phgv2/api/HaplotypeGraphTest.kt

```diff
@@ -45,8 +44,12 @@ class HaplotypeGraphTest {
             }
         }
 
-        Sizeof.printMemoryUse()
+    }
 
+    @Test
+    fun testBadInputHaplotypeGraph() {
+        val graph = HaplotypeGraph(listOf(TestExtension.smallseqLineAHvcfFileBadAltTag))
+        check(graph == null || graph.numberOfRanges() == 0) { "graph not null or empty" }
```

Comment:
You should use an assert here instead of just a check.

Reply:
Ok, changed this